bindings/nodejs/examples/1-create-account.js

```diff
@@ -13,9 +13,14 @@ async function run() {
 
         const account = await manager.createAccount({
             alias: 'Alice',
-            coinType: CoinType.Shimmer,
+            coinType: CoinType.IOTA,
```

Comment:
Maybe add a comment here, that it only needs to be set for the first account?

Reply:
I added it here and on the AccountManager as well